### ISSUE TITLE
acts-as-taggable-on missing from gemspec?

### ISSUE THREAD

**Issue by mfilej** [Bodacious/blogit#46]
I'm using blogit on rails 4, so I bundled the gem from the `rails4` branch. After finishing the installation instructions, I encountered this error:

```
undefined local variable or method `acts_as_taggable' for #<Class:0x007fd01890a728>
```

I'm assuming this is because 10bce41fb1a7e04c9f9cb0d982bc77a462ce47db added the acts-as-taggable-on dependency to the Gemfile instead of the gemspec?

I was able to get the app working by adding this to my own Gemfile:

```
gem "acts-as-taggable-on", github: "mbleigh/acts-as-taggable-on"
```

I'd be happy to submit a pull request if someone can confirm that updating the gemspec is the proper solution.

**Comment by mfilej** [comment]
I guess adding the dependency to the gemspec is not desired if tagging is supposed to be optional. In this case, the `acts_as_taggable` macro could be called conditionally in the model. I'm willing to submit a PR.

**Comment by ghost** [comment]
:+1: for a PR to resolve this issue.  The `acts_as_taggable` dependency should be included in the gemspec shouldn't it?

**Comment by Bodacious** [comment]
Hi guys - sorry for taking so long to get back to you. I work on Blogit in my spare time which is non-existent at the moment.

I removed acts_as_taggable in an earlier version because, for a short while, it wasn't compatible with Rails 4. 
Now that they're up to date, it should be added back to the Gemfile

**Comment by antarr** [comment]
I"m still getting this error in the rails4 branch

**Comment by ghost** [comment]
Same here.

**Comment by Bodacious** [comment]
Fixed in version 1.0.0
